### DISCUSSION TITLE
How do I get Iris/will it support Lunar Client?

### DISCUSSION THREAD

**Discussion by buzzdoesnotbuzz** [IrisShaders/Iris#1114] [Q&A]
I don't use the original Minecraft launcher whether I'm playing on 1.8.9 or whatever the latest version is (1.18.1). I use Lunar Client whenever I'm playing Minecraft, but I also use shaders, and Iris would give me a good performance boost, but there doesn't seem to be a way to get it working. I tried a few different things, but I don't think anything's gonna work unless I'm missing something, or Iris isn't supported on Lunar. If there is a way to get it working, could someone tell me please?

BTW, the few things I tried were changing the directory of Lunar to where the executable jar file is in 1.18 after I set it up for the regular Minecraft launcher, but that didn't work. Well at least I don't think. The video settings menu was the same but idk if it gave me the performance boost with the shaders.

Feel free to ask any questions if I wasn't clear or if you have a suggestion for anything I should do/try.

**Comment by IMS212** [comment]
Lunar is very locked down, and doesn’t allow any mods other than what it gives. We only support Fabric, due to being able to choose all your mods.

**Comment by IMS212** [comment]
Lunar now supports Iris.

**Reply by SohamDasBiswas** [reply]
how can i apply iris to Lunar?
